image-build-to-ecr.yaml

```diff
@@ -0,0 +1,40 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: concourse/docker-image-resource
+    tag: 'pr-227'
```

Comment:
hasn't this PR been merged ?

Reply:
Not sure, was from an existing task but will check it out

Reply:
Yep, removed tag